build.sbt

```diff
@@ -356,6 +356,8 @@ lazy val scalalib =
         IO.copyDirectory(file(s"scalalib/overrides-$epoch.$major/scala"),
                          file("scalalib/src/main/scala/scala"),
                          overwrite = true)
+
+        (file("scalalib/src/main/scala") ** "*.java").get.foreach(IO.delete)
```

Comment:
Removes all the java code from our build of scala library. This fixes conflict situations when we have a Scala-defined override for something that used to be in Java before.